AWSAuthSDK/Sources/AWSMobileClient/Internal/AWSCognitoIdentityUserPool+Extension.h

```diff
@@ -17,4 +17,10 @@
 
 @end
 
+@protocol AWSCognitoUserPoolInternalDelegate
```

Comment:
As discussed, see if we can make this visibility "Project" rather than "Public"

Reply:
Did not work with Project. Keeping as public for now